_projects/armv7a9-zynq7000-zedboard/build.project

```diff
@@ -36,7 +37,7 @@ USER_SCRIPT=(
 	"app ${BOOT_DEVICE} -x dummyfs;-N;devfs;-D ddr ddr"
 	"app ${BOOT_DEVICE} -x zynq7000-uart ddr ddr"
 	"app ${BOOT_DEVICE} -x psh;-i;/etc/rc.psh ddr ddr"
-	"app ${BOOT_DEVICE} -x zynq7000-flash;-r;/dev/mtd1:$((FS_OFFS-0x20000)):0x1000000:jffs2;-p;/dev/mtd1:0x1800000:0x4e0000 ddr ddr"
+	"app ${BOOT_DEVICE} -x zynq7000-flash;-r;/dev/mtd1:$((FS_OFFS-0x20000)):$((FS_SZ)):jffs2;-p;/dev/mtd1:0x1800000:0x4e0000 ddr ddr"
```

Comment:
`${}` is enough, You're not doing any math here

Reply:
done